<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IngresoMedico</class>
 <widget class="QDialog" name="IngresoMedico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>546</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registrar médico</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	
	background-color: rgb(197, 197, 197);
}
QLineEdit{
	border: 1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 35px;
}

QDateEdit{
	border: 1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 31px;
}

QComboBox{
	border: 2px solid white;
	border-radius: 8px;
	padding: 1px 18px 1px 3px;
	color: black;
	height: 35px;
	padding-left: 15px;
	font-weight: bold;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registro de Médico</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIngresarMedico">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(0, 170, 0);
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelarIngreso">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(144, 144, 144);
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rut:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombres">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtApellidos">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Apellidos:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRut">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nombres:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Género:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbGeneroIngresar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Seleccionar Género</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Femenino</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Teléfono:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNumero">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cesfam:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCesfamMedico">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Seleccionar Cesfam</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Santa Julia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Padre Hurtado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Felix de Amesti</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="txtCorreo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Correo:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
